### ISSUE TITLE
Spin alignment test with a₂

### ISSUE THREAD

**Issue by shenvitor** [ComPWA/gluex-nstar#4]
- See also https://github.com/ComPWA/gluex-nstar/issues/97

**Comment by shenvitor** [comment]
@redeboer 
Maybe this issue means later I should include the use of [`ampform-dpd](https://github.com/ComPWA/ampform-dpd)` ?
If so, or this is a separate issues, then maybe I should create a new issue.

**Comment by redeboer** [comment]
Yeah probably better to use ampform-dpd. But are you using two subsystems or only the subsystem that has a2?

**Comment by shenvitor** [comment]
> Yeah probably better to use ampform-dpd. But are you using two subsystems or only the subsystem that has a2?

As we saw [previously](https://github.com/ComPWA/gluex-nstar/issues/5): only one subsystem ($\eta \pi^0$: two mesons) can have $a_2$ resonance, the other two subsystems ($\pi p$ and $\eta p$: a meson and a baryon) can't have $a_2$.
So I guess the subsystem that has $a_2$?

Actually, I am wondering if we should also do this to the two subsystems with N* or Delta too maybe? 🤔

**Comment by redeboer** [comment]
Dalitz-Plot Decomposition is only needed if there are multiple subsystems. $\Delta$ with N would be an example, but for the channel with a2 you can also include another resonance that lives in another subsystem (maybe $\Delta$ or N actually, but thought you were studying those in a different final state).

**Comment by redeboer** [comment]
Closing administratively as not planned, consistent with the decision in #97. The ηπ⁰p DPD follow-up is not currently in scope; the priority remains the ΛKπ model and the unresolved production mechanism in #58.
